public/src/app.js

```diff
@@ -864,4 +864,68 @@ app.cacheBuster = null;
 
 		document.head.appendChild(linkEl);
 	};
+
+	app.updateTags = function () {
+		var metaWhitelist = ['title', 'description', /og:.+/, /article:.+/].map(function (val) {
+			return new RegExp(val);
+		});
+		var linkWhitelist = ['canonical', 'alternate', 'up'];
```

Comment:
A whitelist is used here because not every meta or link tags needs to be updated (especially things like stylesheets).